front/app/src/Views/GameCanvas/GameCanvas.tsx

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable max-lines-per-function */
+/* eslint-disable max-statements */
+import { useEffect, useRef } from "react";
+
+// if we don't import all objects, the game will not work
+import Game from "./objects/Game";
+import Board from "./objects/Board";
+import Ball from "./objects/Ball";
+import Net from "./objects/Net";
+import Player from "./objects/Player";
+import Racket from "./objects/Racket";
+import Position from "./objects/Position";
+import Dimension from "./objects/Dimension";
+
+const	GameCanvas = () =>
+{
+	const game = new Game();
+	game.board.game = game;
+
+	game.ball.game = game;
+	game.net.game = game;
+
+	const	canvasRef = useRef<HTMLCanvasElement>(null);
+	game.board.canvasRef = canvasRef;
+	const	update = () =>
+	{
+		game.playerOne.updatePlayerPosition();
+		game.playerTwo.updatePlayerPosition();
+		game.ball.update();
+	};
+
+	const clear = () =>
+	{
+		if (game.board.ctx)
+		{
+			game.board.ctx.fillStyle = "#fff";
+			game.board.ctx?.clearRect(0, 0,
+				game.board.dim.width, game.board.dim.height);
+		}
+	};
+
```

Comment:
Here the type of the variable is KeyboardEvent

Reply:
Done.